blueprints/instance-initializer-test/files/tests/unit/instance-initializers/__name__-test.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+import { initialize } from '<%= dependencyDepth %>/instance-initializers/<%= dasherizedModuleName %>';
+import { module, test } from 'qunit';
+
+var appInstance, application;
+
+module('<%= friendlyTestName %>', {
+  beforeEach: function() {
+    Ember.run(function() {
+      application = Ember.Application.create();
+      appInstance = application.__deprecatedInstance__;
+      application.deferReadiness();
```

Comment:
You should be able to call `app.visit()` which returns a promise that resolves to the app instance.